Documentation/helm-operator.md

```diff
@@ -196,8 +168,6 @@ helm install --namespace rook-ceph rook-ceph rook-release/rook-ceph --set rbacEn
 
```

Comment:
Please remove the following line 161 as well:
> For Helm `v2.x` the `--name` flag must be specified instead of just `rook-ceph`, e.g., `--name rook-ceph`.

Reply:
done